jsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "vueCompilerOptions": {
     "target": 2.7
+  },
+  "compilerOptions": {
+    "strictNullChecks": true
```

Comment:
This makes vscode actually show `undefined` and `null` in type definitions e.g. auto-complete and the popup that appears when you hover over functions.